persistent/Database/Persist/Quasi/Internal.hs

```diff
@@ -1433,21 +1442,30 @@ takeForeign ps entityName = takeRefTable
                                     , show plen, " parent fields"
                                     ]
 
-        go ((parseCascadeAction CascadeDelete -> Just cascadingAction) : rest) onDelete' onUpdate =
+        go ((parseSchemaName -> Just schemaName) : rest) schemaName' onDelete onUpdate
+            | isJust schemaName' = error $ errorPrefix ++ "found more than one schema definition"
+            | otherwise = go rest (Just schemaName) onDelete onUpdate
```

Comment:
Do you mind changing this `go` to accord with the style of the rest of the `go` definitions? I.e., instead of using guards, start with `case schemaName' of`.

Reply:
(The contributor guide mentions that we should conform to the existing style in a module.)

Reply:
it turns out we shouldn't need this at all